consensus/service/src/api/grpc_error.rs

```diff
@@ -139,6 +139,7 @@ impl From<ConsensusGrpcError> for Result<ProposeTxResponse, RpcStatus> {
         match src {
             ConsensusGrpcError::TransactionValidation(err) => {
                 let mut resp = ProposeTxResponse::new();
+                resp.set_err_msg(err.to_string());
```

Comment:
this is a line that I failed to add to an earlier PR: https://github.com/mobilecoinfoundation/mobilecoin/pull/2501